package.json

```diff
@@ -27,7 +27,7 @@
     "browser"
   ],
   "optionalDependencies": {
-    "default-browser-id": "1.0.2"
+    "default-browser-id": "1.0.4"
```

Comment:
I used to be all for strict versions but I am changing my mind a bit, can you use `~1.0.4` instead?